requirements.txt

```diff
@@ -6,4 +6,5 @@ dominate>=2.3.1
 dill
 tensorboardX
 scipy
-opencv-python
\ No newline at end of file
+opencv-python
+torchvision
```

Comment:
For what purpose we need this package here?

Reply:
Because without it did not work, at least when I have tried to install locally.